server/src/main/kotlin/org/covid19support/modules/users/model.kt

```diff
@@ -69,7 +70,7 @@ object Users : IntIdTable("users") {
             it[first_name] = user.firstName
             it[last_name] = user.lastName
             it[description] = user.description
-            it[is_instructor] = user.isInstructor
+            it[is_instructor] = user.isInstructor!!
```

Comment:
I feel like these exclamation marks are forcing something and that may be a bad thing down the road.